app/views/publish/courses/confirmation.html.erb

```diff
@@ -173,14 +177,16 @@
           <% end %>
         <% end %>
 
-        <% if course.school_direct_salaried_training_programme? || course.pg_teaching_apprenticeship? %>
+        <% if course.school_direct_salaried_training_programme? || course.pg_teaching_apprenticeship? || course.teacher_degree_apprenticeship? %>
```

Comment:
Is what makes these three things the same that they are salaried? or rather, do not have fees? A comment, or a descriptive method that returns the required courses might help future us know why we have this conditional

Reply:
Done